server/service/src/rnr_form/generate_rnr_form_lines.rs

```diff
@@ -123,6 +123,7 @@ pub fn generate_rnr_form_lines(
                 id: uuid(),
                 rnr_form_id: rnr_form_id.to_string(),
                 item_id,
+                requisition_id: None,
```

Comment:
perhaps should be `requisition_line_id` to link directly

Reply:
Ohh haha yes it should, thanks! 😅